src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_kds.c

```diff
@@ -400,6 +401,33 @@ static void notify_execbuf(struct kds_command *xcmd, int status)
 	}
 }
 
+static bool is_valid_execbuf(struct xocl_dev *xdev, struct drm_xocl_bo *xobj,
```

Comment:
the function name probably should be cpy_and_validate_execbuf() since you do a copy internally.